.jpmignore

```diff
@@ -0,0 +1,21 @@
+# Ignore these files while packaging extension
+.jpmignore
+*.md
+*.xpi
+.git/
+chrome/content/privly-applications/*.md
```

Comment:
[This](https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/jpmignore) documentation leads me to believe that this line and several others are redundant since `*.[foo]` covers the files in the root or in `chrome/content/privly-applications`.  I have not tested it so I might be reading too much into it.